src/Components/Alerts/Alert.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect } from "react";
+import { Alert, Box, Slide } from "@mui/material";
+import { closeAlert } from "../../Slices/modalSice";
+import { useAppDispatch } from "../../Redux/hooks";
+import "./AlertStyle.css";
+
+type AlertProps = {
+  isShowModalAlertMessage: boolean;
+};
+
+const Alerts = ({ isShowModalAlertMessage }: AlertProps) => {
+  const dispatch = useAppDispatch();
+
+  useEffect(() => {
```

Comment:
what the reason for useEffect without any dependencies?
try to optimize in using isShowModalAlertMessage as dependency